scripts/create-darwin-volume.sh

```diff
@@ -83,6 +61,64 @@ suggest_report_error(){
     echo "       please report this @ https://github.com/nixos/nix/issues" >&2
 }
 
+generate_mount_command(){
+    if test_filevault_in_use; then
+        printf "    <string>%s</string>\n" /bin/sh -c '/usr/bin/security find-generic-password -a "Nix Volume" -w | /usr/sbin/diskutil apfs unlockVolume "Nix Volume" -mountpoint /nix -stdinpassphrase'
+    else
+        printf "    <string>%s</string>\n" /usr/sbin/diskutil mount -mountPoint /nix "Nix Volume"
```

Comment:
If a volume exists starting with "Nix" already, we use that volume. This mount command needs to be parameterized accordingly.

Reply:
Heck, for that matter we should probably embed the service too, so that way if I delete the volume and clean up everything except for my keychain password, when it creates a brand new password this won't accidentally read the old one.

Reply:
> Heck, for that matter we should probably embed the service too

Can you clarify what you mean by "embed the service"? Passing the `-s` arg to find-generic-password?

Reply:
Yeah, the `-s` arg is what I meant.